ui/selectors/selectors.js

```diff
@@ -706,17 +705,117 @@ export function getTargetSubjectMetadata(state, origin) {
   return metadata;
 }
 
+///: BEGIN:ONLY_INCLUDE_IN(snaps)
+/**
+ * Input selector for reusing the same state object.
+ * Used in memoized selectors created with createSelector
+ * when raw state is needed to be passed to other selectors
+ * used to achieve re-usability.
+ *
+ * @param state - Redux state object.
+ * @returns Object - Redux state object.
+ */
+export const rawStateSelector = (state) => state;
+
+/**
+ * Input selector used to retrieve Snaps that are added to Snaps Directory.
+ *
+ * @param state - Redux state object.
+ * @returns Object - Containing verified Snaps from the Directory.
+ */
+const selectVerifiedSnapsRegistry = (state) =>
+  state.metamask.database.verifiedSnaps;
```

Comment:
```suggestion
  state.metamask.database?.verifiedSnaps;
```
I don't think this is safe, `state.metamask.database` is `null` by default.

Reply:
Updated.